cpp-low-level/dapp.c

```diff
@@ -0,0 +1,91 @@
+/* Copyright Cartesi and individual authors (see AUTHORS)
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <fcntl.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
+
+#include "libcmt/rollup.h"
+
+static int finish_request(cmt_rollup_t *me, cmt_rollup_finish_t *finish, bool accept) {
```

Comment:
Code can be simplified a bit since `accept` is always `true`.

```suggestion
static int finish_request(cmt_rollup_t *me, cmt_rollup_finish_t *finish) {
    finish->accept_previous_request = true;
```